tests/acceptance/features/cliTwoFactorTOTP/steps.feature

```diff
@@ -0,0 +1,132 @@
+
+  Scenario: Unverifying multiple users containing one having no secret set should work for other users, but fail
+    Given user "user8" has logged in using the webUI
+    And the user has browsed to the personal security settings page
+    And the user has activated TOTP Second-factor auth but not verified
+    And the administrator has invoked occ command "twofactor_totp:set-secret-verification-status -u user8 true"
+    When the administrator invokes occ command "twofactor_totp:set-secret-verification-status -u user8 -u new-user false"
+    Then the command should have failed with exit code 1
+    And the command output should contain the text "The secret of user8 is unverified"
+    And the command output should contain the text "User has no secret: new-user"
+    And user "user8" should be able to access a skeleton file
+    And user "new-user" should be able to access a skeleton file
+
+  Scenario: Verifying all users that use TOTP should work
+    Given user "user8" has logged in using the webUI
+    And the user has browsed to the personal security settings page
+    And the user has activated TOTP Second-factor auth but not verified
+    When the administrator invokes occ command "twofactor_totp:set-secret-verification-status --all true"
+    Then the command should have been successful
+    And the command output should contain the text "The status of all TOTP secrets has been set to verified"
+    And user "user8" using password "%regularuser%" should not be able to download file "textfile0.txt"
+    And user "new-user" should be able to access a skeleton file
+
+  Scenario: Unverifying all users that use TOTP should work
+    Given user "user8" has logged in using the webUI
+    And the user has browsed to the personal security settings page
+    And the user has activated TOTP Second-factor auth but not verified
+    And the administrator has invoked occ command "twofactor_totp:set-secret-verification-status --all true"
+    When the administrator invokes occ command "twofactor_totp:set-secret-verification-status --all false"
+    Then the command should have been successful
+    And the command output should contain the text "The status of all TOTP secrets has been set to unverified"
+    And user "user8" should be able to access a skeleton file
+    And user "new-user" should be able to access a skeleton file
+
+  @skip @issue-89
+  Scenario: Deleted user having TOTP enabled recreated should not ask for password
+    Given user "user8" has logged in using the webUI
+    And the user has browsed to the personal security settings page
+    And the user has activated TOTP Second-factor auth but not verified
+    And the administrator has invoked occ command "twofactor_totp:set-secret-verification-status --all true"
+    When the administrator deletes user "user8" using the provisioning API
+    And the administrator creates user "user8" using the provisioning API
+    Then user "user8" should be able to access a skeleton file
```

Comment:
Currently, I am not able to fail this scenario after using the patch from #90. So, I'll just skip this, and I'll add a comment to that PR to unskip this.